entrypoint.sh

```diff
@@ -35,9 +30,40 @@ if ! APTIBLE_OUTPUT_FORMAT=json aptible apps | jq -e ".[] | select(.handle == \"
   exit 1
 fi
 
-aptible deploy --environment "$INPUT_ENVIRONMENT" \
-               --app "$INPUT_APP" \
-               --docker-image "$INPUT_DOCKER_IMG" \
-               --private-registry-username "$INPUT_PRIVATE_REGISTRY_USERNAME" \
-               --private-registry-password "$INPUT_PRIVATE_REGISTRY_PASSWORD" \
-               ${INPUT_CONFIG_VARIABLES}
+if [ "$INPUT_TYPE" == "git" ]; then
+  BRANCH="$GITHUB_HEAD_REF"
+  if [ -z "$BRANCH" ]; then
+    BRANCH="$GITHUB_REF_NAME"
+  fi
+  if [ -z "$BRANCH" ]; then
+    echo "Aborting: branch is not set; this shouldn't happen, please contact support"
```

Comment:
you should probably specify Aptible support since it might seem like we're pointing to GHA support

Reply:
Updated!